admin/main.go

```diff
@@ -416,6 +414,9 @@ func main() {
 		}
 	}()
 
+	// multiple listeners channel
+	finish := make(chan bool)
```

Comment:
where is this used?

Reply:
Hmmm I had this from a long time ago when I was running all services in the same binary, but now I am not using it anymore. Remove!